pkg/asset/ignition/bootstrap/baremetal/template.go

```diff
@@ -55,6 +55,9 @@ type TemplateData struct {
 
 	// Hosts is the information needed to create the objects in Ironic.
 	Hosts []*baremetal.Host
+
+	// ProvisioningNetwork displays the type of provisioning network being used
+	ProvisioningNetwork string
```

Comment:
Likewise, here, these changes should not be in this PR.

Reply:
Thanks @staebler for your comments, actually the PR wasn't completed.